scripts/quests/sandoria/Distant_Loyalties.lua

```diff
@@ -0,0 +1,161 @@
+-----------------------------------
+-- Distant Loyalties
+-----------------------------------
+-- !addquest 0 74
+-- Femitte: !gotoid 17719491
+-- Michea: !gotoid 17739820
+-----------------------------------
+
+local quest = Quest:new(xi.questLog.SANDORIA, xi.quest.id.sandoria.DISTANT_LOYALTIES)
+
+quest.reward =
+{
+    fame = 30,
+    fameArea = xi.fameArea.SANDORIA,
+    item = xi.item.WHITE_CAPE,
+}
+
+quest.sections =
+{
+    {
+        check = function(player, status, vars)
+            return status == xi.questStatus.QUEST_AVAILABLE and
+            player:getFameLevel(xi.fameArea.SANDORIA) >= 4
+        end,
+
+        [xi.zone.SOUTHERN_SAN_DORIA] =
+        {
+            ['Femitte'] = quest:progressEvent(663),
+
+            onEventFinish =
+            {
+                [663] = function(player, csid, option, npc)
+                    if option == 0 then
+                        quest:begin(player)
+                        npcUtil.giveKeyItem(player, xi.ki.GOLDSMITHING_ORDER)
+                    end
+                end,
+            },
+        },
+    },
+
+    {
+        check = function(player, status, vars)
+            return status == xi.questStatus.QUEST_ACCEPTED
+        end,
+
+        [xi.zone.SOUTHERN_SAN_DORIA] =
+        {
+            ['Femitte'] =
+            {
+                onTrigger = function(player, npc)
+                    local questProgress = quest:getVar(player, 'Prog')
```

Comment:
No need to define this as a var here, since its only used in one of the conditions